src/init.jl

```
@@ -32,11 +32,18 @@ const read_stdout = Ref{Base.PipeEndpoint}()
 const read_stderr = Ref{Base.PipeEndpoint}()
 const socket_locks = Dict{Socket,ReentrantLock}()
 
+connection_file = ""
+
+function qtconsole()
+    spawn(`$jupyter qtconsole --existing $connection_file`)
```

Probably should throw an error if `isempty(connection_file)` is `true`